### PR TITLE
Added build script for httpmime

### PR DESCRIPTION
**Pull request by saurabh-gore**
- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?

Package Name : httpmime
Package Version : rel/v4.1.2
Package URL : [httpmime](https://github.com/apache/httpcomponents-client)

Tested On : UBI 8.5

### CODE REVIEW

**Review by Siddhesh-Ghadi — CHANGES_REQUESTED** [review]
Please squash the commit. Rest all looks good.

File: h/httpmime/httpmime_ubi_8.5.sh [hunk 1]
```diff
@@ -0,0 +1,45 @@
+#! /bin/bash
+# -----------------------------------------------------------------------------
+# Package		: httpmime
+# Version		: rel/v4.1.2
+# Source repo	: https://github.com/apache/httpcomponents-client
+# Tested on		: "Red Hat Enterprise Linux 8.5" (Docker)
```
Comment (Siddhesh-Ghadi) [inline h/httpmime/httpmime_ubi_8.5.sh:6]: Can we change this to UBI?
```suggestion
# Tested on		: UBI 8.5
```

**Comment by saurabh-gore** [comment]
@Siddhesh-Ghadi Updated , Please review.

**Review by Siddhesh-Ghadi — APPROVED** [review]
LGTM.

**Pull request merged**